.env

```diff
@@ -18,7 +18,7 @@ COGMENT_VERSE_MLFLOW_PORT=5000
 COGMENT_VERSE_GRAFANA_PORT=5001
 COGMENT_VERSE_PROMETHEUS_PORT=5002
 COGMENT_VERSE_GRPCWEBPROXY_PORT=5003
-COGMENT_VERSE_WEBCLIENT_PORT=5004
+COGMENT_VERSE_WEBCLIENT_PORT=8080
```

Comment:
Not a good idea they are different to avoid hiding dumb bugs like misused environment variable

Reply:
They're like this to have the sample port dockerized and undockerized. As both these use cases are equally as used.

I can absolutely separate them and change the readme though

Reply:
Undockerized is more convenient for development, right? Does `cogment run dev` make that obsolete? And if not, we could change it so that it also supports webclient development.

Reply:
I don't see any real harm in leaving them as the same port

Reply:
I've been burned before mixing environment variables in code and only finding the issue when I deployed later

Reply:
Done